app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt

```diff
@@ -178,6 +187,9 @@ class DashboardActivity : MifosBaseActivity(), View.OnClickListener, NavigationV
             R.id.item_about_us -> {
                 showAboutUs()
             }
+            R.id.item_settings -> {
+                replaceFragment(SettingsFragment.newInstance(), true, R.id.container)
+            }
```

Comment:
@PiyushSharma99 please remove your previous change for the settings fragment from this PR.

Reply:
Done @jddeep Thanks!